packages/core/useDraggable/component.ts

```diff
@@ -38,8 +38,8 @@ export const UseDraggable = /* #__PURE__ */ defineComponent<UseDraggableProps>({
     'containerElement',
   ] as unknown as undefined,
   setup(props, { slots }) {
-    const target = shallowRef()
-    const handle = computed(() => props.handle ?? target.value)
+    const target = shallowRef<HTMLElement | SVGElement | null>()
```

Comment:
do we need `null` here?

Reply:
if yes, why not here:
https://github.com/vueuse/vueuse/pull/4761/files#diff-f6eea9455a372b89e76b87072ff29e2d5710f38c2b2100d276216b4ee45856b4R11